lib/model/autocomplete.dart

```diff
@@ -192,10 +192,13 @@ class MentionAutocompleteView extends ChangeNotifier {
   final PerAccountStore store;
   final Narrow narrow;
 
+  MentionAutocompleteQuery? get query => _currentQuery;
   MentionAutocompleteQuery? _currentQuery;
```

Comment:
Also let's rename the backing field to match the getter and setter: so just `_query`. (Probably should have written it that way in the first place, to match the setter.)